doc/tutorials/using_industry_scenario_on_nuc.rst

```diff
@@ -0,0 +1,105 @@
+.. _using_industry_scenario_on_nuc:
+
+Launch Two Guest VMs on NUC Using Industry Scenario
+###################################################
+
+ACRN hypervisor supports new industry scenario
+since the tag ``acrn-2019w18.7-140000p``. In industry scenario,
+up to three Guest VMs (e.g. Vxworks + RTVM + WaaG) can be launched from
```

Comment:
It is better to replace WaaG with Windows.

Reply:
Fixed.